README.md

```diff
@@ -57,6 +57,10 @@ Note that the external-provisioner does not scale with more replicas. Only one e
 
 * `--worker-threads <num>`: Number of simultaneously running `ControllerCreateVolume` and `ControllerDeleteVolume` operations. Default value is `100`.
 
+* `--kube-api-qps <num>`: QPS for clients that communicate with the kubernetes apiserver. Defaults to `5.0`.
+
+* `--kube-api-burst <num>`: Burst for clients that communicate with the kubernetes apiserver. Defaults to `10`.
```

Comment:
For someone who is not familiar with throttling, is it possible to explain the meaning of "QPS" and "burst", perhaps by linking to a document which explain these concepts?

Nit: s/kubernetes/Kubernetes/

Reply:
https://github.com/kubernetes-csi/external-provisioner/pull/461